ansible/roles/tldraw-client-core/tasks/main.yml

```
@@ -0,0 +1,24 @@
+- name: TldrawClientService
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: "{{ NAMESPACE }}"
+    template: tldraw-client-svc.yml.j2
+
+- name: TldrawClientConfigmap
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: "{{ NAMESPACE }}"
+    template: tldraw-client-configmap.yml.j2
+    apply: yes
+
+- name: TldrawClientDeployment
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: "{{ NAMESPACE }}"
+    template: tldraw-client-deployment.yml.j2
+
+- name: TldrawClientIngress
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: "{{ NAMESPACE }}"
+    template: tldraw-client-ingress.yml.j2
```

so let us **apply** the new ingress correctly.
Before it was an update with merge and the removed labels were still there. 
```suggestion
    template: tldraw-client-ingress.yml.j2
    apply: yes

```